packages/dcos-integration-test/extra/test_networking.py

```diff
@@ -1,37 +1,156 @@
+import contextlib
+import json
 import logging
+import random
+import threading
+from collections import deque
 from subprocess import check_output
 
 import pytest
-
 import requests
-
 import retrying
 
 from pkgpanda.build import load_json
 from test_util.marathon import get_test_app, get_test_app_in_docker
 
 
+TIMEOUT = 500
+
+
 def lb_enabled():
     config = load_json('/opt/mesosphere/etc/expanded.config.json')
     return config['enable_lb'] == 'true'
 
 
-def ensure_routable(cmd, service_points, timeout=120):
-    @retrying.retry(wait_fixed=2000,
-                    stop_max_delay=timeout * 1000,
-                    retry_on_result=lambda ret: ret is False,
-                    retry_on_exception=lambda x: True)
-    def _ensure_routable():
-        proxy_uri = 'http://{}:{}/run_cmd'.format(service_points[0].host, service_points[0].port)
-        logging.info('Sending {} data: {}'.format(proxy_uri, cmd))
-        r = requests.post(proxy_uri, data=cmd)
-        logging.info('Requests Response: %s', repr(r.json()))
-        assert r.json()['status'] == 0
-    return _ensure_routable
+@retrying.retry(wait_fixed=2000,
+                stop_max_delay=TIMEOUT * 1000,
```

Comment:
Timeout has changed from 2 minutes to 8 minutes and 20 seconds. This seems like a pretty big change. If we need to bump it, maybe 5 minutes (300) would be a more suitable value?